vars/kibanaCoverage.groovy

```diff
@@ -148,9 +148,10 @@ def generateReports(title) {
     cd ..
     . src/dev/code_coverage/shell_scripts/extract_archives.sh
     . src/dev/code_coverage/shell_scripts/fix_html_reports_parallel.sh
-    . src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh
-    # zip combined reports
-    tar -czf kibana-coverage.tar.gz target/kibana-coverage/**/*
+    . src/dev/code_coverage/shell_scripts/merge_functional.sh
+    . src/dev/code_coverage/shell_scripts/copy_jest_report.sh
+    # zip functional combined report
+    tar -czf kibana-functional-coverage.tar.gz target/kibana-coverage/functional-combined/*
```

Comment:
We only merge functional tests now, but we still need to copy jest for ingestion.  Functional combined reported is uploaded separately since jest was already in artefacts after tests execution.